registry/sql-registry/registry/db_registry.py

```diff
@@ -52,8 +52,10 @@ class DbRegistry(Registry):
     def __init__(self):
         self.conn = connect()
         if os.environ.get("FEATHR_SANDBOX"):
-            if os.environ.get("FEATHR_SANDBOX_REGISTRY_URL"):
-                engine = db.create_engine(f'sqlite:///{os.environ.get("FEATHR_SANDBOX_REGISTRY_URL")}?check_same_thread=False')
+            sandbox_registry_url = os.environ.get("FEATHR_SANDBOX_REGISTRY_URL")
```

Comment:
@enya-yx , based on our work on code coverage increase, will it be useful to add some "FEATHR_SANDBOX" specific test functions?

Reply:
> @enya-yx , based on our work on code coverage increase, will it be useful to add some "FEATHR_SANDBOX" specific test functions?

Agree with that. To maintain the test coverage ratio and also provide some examples as a reference, I suggest to add some related test cases.

Reply:
This is good point, unfortunatelly those code cannot be unit tested since it requires a docker env setup. 

For instructions, it's part of the readme file:

By default, Feathr Sandbox uses a SQLite backend in Feathr registry, and the content will be deleted if you restart the container. If you want to use a persistent registry, say a remote MySQL database or SQL Server, you can configure an environment variable when starting the Feathr Sandbox container like below, by specifying `FEATHR_SANDBOX_REGISTRY_URL` environment variable:

```bash
docker run -it --rm -p 8888:8888  -p 8000:8000 -p 8081:80 -p 8080:8080 -p 7080:7080 --env API_BASE="api/v1" --env FEATHR_SANDBOX=True --env FEATHR_SANDBOX_REGISTRY_URL="mysql://scott:tiger@localhost/foo" -e GRANT_SUDO=yes feathrfeaturestore/feathr-sandbox
```

Take a look at the [Database URLs](https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls) section for more details on how this URL is formatted. Basically it will be like this: